python/lsst/dax/obscore/obscore_exporter.py

```diff
@@ -344,12 +344,12 @@ def to_votable(self, limit: int | None = None) -> astropy.io.votable.tree.VOTabl
                 # This must be a non-standard field. Attempt to add it.
                 kwargs = {}
                 datatype = ""
-                if (
-                    arrow_field.type.equals(pyarrow.int64())
-                    or arrow_field.type.equals(pyarrow.int32())
-                    or arrow_field.type.equals(pyarrow.int16())
-                ):
+                if arrow_field.type.equals(pyarrow.int64()):
```

Comment:
Yes, best to preserve types under the assumption that the upstream code is setting them as desired.